Add  mediastack scraper script and output file

DilipBChhetri opened the pull request:
Added a scraper script for the Mediastack API and its output file.  
The script fetches and stores news data in JSON format for further analysis.

DilipBChhetri committed:
Add  mediastack scraper script and output file

meriwhoo commented on scrape/mediastack/scrappingfinal.py:
```diff
@@ -0,0 +1,98 @@
+                    "published_date": a.get("published_at"),
+                    "url": a.get("url"),
+                    "author": a.get("author"),
+                    "content": a.get("content"),
```
The content scraped from the article seems to be null for all.

meriwhoo reviewed (changes requested):
Fix for correctly scraping the content from the articles.

DilipBChhetri committed:
Update scrappingfinal.py

DilipBChhetri committed:
Add files via upload

Implemented a dual-method web scraper that first extracts the author and full article content using the newspaper3k library. If newspaper3k fails to retrieve the data (due to unsupported site structures or errors), the scraper automatically falls back to using Playwright for dynamic scraping. If both methods fail, the scraper skips the article to ensure the pipeline continues smoothly. This approach enhances reliability when dealing with diverse and complex news websites.
